templates/init_webservice.sh.template

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
+# Make the script's directory the working directory, which is also where
+the webservice JAR is.
```

Comment:
think this is splitting onto a new line, making an invalid command

Reply:
Yikes, how embarrassing.

Reply:
No worries, looks like the first one is fixed, this one isn't